### PR TITLE
Miscellaneous improvements

### PR DESCRIPTION
**Pull request by tjanez**
I think it will be easier to review these separately, so I extracted them from my #2300 PR.

### CODE REVIEW

File: go/worker/common/worker.go [hunk 1]
```diff
@@ -339,7 +346,7 @@ func New(
 ) (*Worker, error) {
 	cfg, err := newConfig()
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "worker/common: failed to initialize config")
```
Comment (kostko) [inline go/worker/common/worker.go]: For new code, no need to use the external errors package (see #2057):
```suggestion
		return nil, fmt.Errorf("worker/common: failed to initialize config: %w", err)
```

File: go/oasis-test-runner/oasis/validator.go [hunk 2]
```diff
@@ -99,7 +101,7 @@ func (net *Network) NewValidator(cfg *ValidatorCfg) (*Validator, error) {
 		"registry", "node", "init",
 		"--datadir", val.dir.String(),
 		"--node.consensus_address", fmt.Sprintf("127.0.0.1:%d", val.consensusPort),
-		"--node.expiration", "1000000",
+		"--node.expiration", "1",
```
Comment (Yawning) [inline go/oasis-test-runner/oasis/validator.go]: At one point, this would break horribly if it wasn't at least "2", but that might have been fixed.

Comment (tjanez) [reply go/oasis-test-runner/oasis/validator.go]: Yes, @kostko, told me that it should be `1` or `2`.

I tried with `1` and the tests passed :slightly_smiling_face:.

Comment (kostko) [reply go/oasis-test-runner/oasis/validator.go]: Validators expiring seems really dangerous though. I think the scheduler will currently even allow having only a single validator which could then do arbitrary stuff?

Comment (Yawning) [reply go/oasis-test-runner/oasis/validator.go]: > I think the scheduler will currently even allow having only a single validator which could then do arbitrary stuff?

Correct.  However:
 * We have not established the minimum number of validators that we want (and it is a policy decision that I am not qualified to make).  The correct thing to do is for someone to come up with validator committee minimum parameters, that we enforce.
 * Even with such parameters, the only safe behavior in the event that we do not have a sufficient node pool to elect a validator committee is to halt.

Comment (kostko) [reply go/oasis-test-runner/oasis/validator.go]: Agreed, https://github.com/oasislabs/oasis-core/issues/2314.

**Pull request merged**